plugins/TwoFactorAuth/lang/en.json

```diff
@@ -2,50 +2,50 @@
+        "InvalidAuthCode": "Incorrect two-factor authentication code.",
+        "RequiredAuthCodeNotConfiguredAPI": "You must set up two-factor authentication. Please log in to do so.",
+        "VerifyIdentifyExplanation": "Verify yourself by opening the two-factor authentication app on your device to get your authentication code.",
+        "DontHaveYourMobileDevice": "Don't have your mobile device?",
         "EnterRecoveryCodeInstead": "Enter one of your recovery codes",
-        "AskSuperUserResetAuthenticationCode": "Ask super user to reset your authentication code",
+        "AskSuperUserResetAuthenticationCode": "Ask a superuser to reset your authentication code",
         "SetupIntroFollowSteps": "Please follow these steps to set up two-factor authentication:",
-        "SetupFinishedTitle": "Congratulations! Your account is now more secure.",
-        "SetupFinishedSubtitle": "You have successfully set up two-factor authentication. Next time you log in, you will need to also enter the authentication code. Make sure you have your mobile device or your backup codes with you.",
-        "WarningChangingConfiguredDevice": "You are about to change the configured two-factor authentication device. This will invalidate any previously configured device.",
+        "SetupFinishedTitle": "Your account is now more secure.",
+        "SetupFinishedSubtitle": "You have now set up two-factor authentication. From now on you nee to enter an authentication code to log in. Make sure you have your mobile device or your backup codes with you.",
+        "WarningChangingConfiguredDevice": "You are about to change the configured two-factor authentication device. This will invalidate any previously configured devices.",
         "ShowRecoveryCodes": "Show recovery codes",
         "ConfirmSetup": "Confirm setup",
-        "NotPossibleToLogIn": "Cannot log in to Matomo Analytics",
-        "LostAuthenticationDevice": "Hi,%1$sI have two-factor authentication enabled and lost my authentication device. Could you please reset two-factor authentication for my username %5$s? You can find the instructions for this here: %6$s.%2$sThe Matomo URL is %3$s.%4$sThanks",
-        "WrongAuthCodeTryAgain": "Wrong authentication code entered. Please try again.",
-        "DisableTwoFA": "Disable two-factor authentication",
-        "EnableTwoFA": "Enable two-factor authentication",
-        "ConfirmDisableTwoFA": "Are you sure you want to disable two-factor authentication for your account? Having two-factor authentication enabled increases your account security.",
-        "VerifyAuthCodeIntro": "Please enter the six-digit code from your authenticator app below to confirm you have successfully set up on your device.",
-        "VerifyAuthCodeHelp": "Please enter the six-digit code that has been generated on your mobile device after scanning the bar code.",
+        "NotPossibleToLogIn": "Cannot log in",
```

Comment:
This is used as a subject of an email. So might be better to keep the ` to Matomo Analytics`